### DISCUSSION TITLE
What is accesspoint password?

### DISCUSSION THREAD

**Discussion by Maverick565** [rern/rAudio#1636] [General]
Where i can find accesspoint password?

**Comment by cmh714** [comment]
It tells you when you enable it....raudioap

**Reply by Maverick565** [reply]
But i don't connect it to hdmi or ethernet. Oy wifi because it's Rpi zero 2w.

**Reply by EarsofTin** [reply]
Maybe it could be added to the FAQ.  A recent power outage disabled Wi-Fi on my RPi and I could not connect to the access point without the unknown to me password. Had to disconnect RPi from my audio system and connect via wire.  On a new set up the user needs to know the password to connect via accesspoint and then set up wifi.  I believe the password is a relatively new feature.

**Comment by Maverick565** [comment]
But i don't connect it to hdmi or ethernet. Oy wifi because it's Rpi zero
2w.

W dniu czwartek, 21 marca 2024 cmh714 ***@***.***> napisał(a):
